src/relentless/simulate/hoomd.py

```diff
@@ -1803,54 +1374,24 @@ def __init__(self, initializer, operations=None, device="auto"):
         if not _hoomd_found:
             raise ImportError("HOOMD not found.")
 
-        if (
-            _hoomd_version.major == 2
-            and packaging.version.Version(numpy.__version__).major >= 2
-        ):
-            warnings.warn(
-                "NumPy 2 is likely incompatible with HOOMD 2, "
-                "suggest to downgrade to numpy<2.",
-                RuntimeWarning,
-            )
+        if _hoomd_version.major < 3:
```

Comment:
Check the docstring for this class, and make sure it documents the minimum version correctly.